py-polars/requirements-ci.txt

```diff
@@ -0,0 +1,4 @@
+# ---------------------------------------------------
+# Packages that we require unit tests that run on CI
+# ---------------------------------------------------
+torch
```

Comment:
Can we point this one to `--index-url https://download.pytorch.org/whl/cpu`.

That should reduce the download time a lot, if I am not mistaken.

Reply:
Done; let's see if it trims things down further  ✌️